src/libraries/System.ComponentModel.Annotations/src/System/ComponentModel/DataAnnotations/AssociatedMetadataTypeTypeDescriptor.cs

```diff
@@ -1,6 +1,9 @@
 // Licensed to the .NET Foundation under one or more agreements.
 // The .NET Foundation licenses this file to you under the MIT license.
 
+// TODO: Enable after System.ComponentModel.TypeDescriptionProvider is annotated
+#nullable disable
+
```

Comment:
should `AssociatedMetadataType` be nullable? I'm seeing null check in `GetAttributes()`

Reply:
Definitely seems so - I left AssociatedMetadataTypeTypeDescriptor and AssociatedMetadataTypeTypeDescriptionProvider unannotated (and disabled) because of the dependency on System.ComponentModel.TypeDescriptionProvider. The idea is to return to these soon once that's done.